packages/quantic/force-app/main/default/lwc/quanticNotifications/quanticNotifications.js

```diff
@@ -68,13 +88,30 @@ export default class QuanticNotifications extends LightningElement {
     );
   }
 
-  get notifications() {
-    return (
-      this.notifyTriggerState?.notifications.map((notification, index) => ({
-        value: notification,
-        id: index,
-      })) ?? []
-    );
+  handleSearchStatusChange() {
+    if (
+      this.searchStatus?.state?.isLoading ||
+      this.searchStatus?.state?.hasError
+    ) {
+      this.notifications = [];
+    } else {
+      this.notifications =
+        this.notifyTrigger?.state?.notifications.map((notification, index) => ({
+          value: notification,
+          id: index,
+          visible: true,
+        })) ?? [];
+    }
```

Comment:
why notifications are set in this subscriber function and not in the notifications subscriber?

For me that's the logic that I see for example in plain english:
- Subscribe to the notifications state and update `this.notifications` each time the state changes.
- Subscribe to search status state to reset the visibility to true when a new search is detected and empty the notifications when an error occurs.

this would be translated to this:
```javascript
  updateState() {
    this.notifications =
      this.notifyTrigger?.state?.notifications.map((notification, index) => ({
        value: notification,
        id: index,
        visible: true,
      })) ?? [];
  }

  handleSearchStatusChange() {
    if (!this.searchStatus?.state.isLoading) {
      if (this.searchStatus?.state?.hasError) {
        this.notifications = [];
      } else {
        this.notifications =
          this.notifications?.map((notification) => ({
            ...notification,
            visible: true,
          })) ?? [];
      }
    }
  }
```